src/server.h

```diff
@@ -84,6 +84,10 @@ typedef long long ustime_t; /* microsecond time type. */
 #include "endianconv.h"
 #include "crc64.h"
 
+/* min/max */
+#define min(a, b) (a) < (b) ? a : b
+#define max(a, b) (a) > (b) ? a : b
```

Comment:
```suggestion
#define min(a, b) ((a) < (b) ? (a) : (b))
#define max(a, b) ((a) > (b) ? (a) : (b))
```

Would this be more safe.